browser/playlist/playlist_tab_helper.cc

```diff
@@ -53,6 +54,28 @@ void PlaylistTabHelper::RemoveObserver(PlaylistTabHelperObserver* observer) {
   observers_.RemoveObserver(observer);
 }
 
+void PlaylistTabHelper::AddItems(std::vector<mojom::PlaylistItemPtr> items) {
+  DCHECK(!is_adding_items_);
+  DCHECK(items.size());
+  is_adding_items_ = true;
+
+  auto* contents = web_contents();
+  DCHECK(contents);
```

Comment:
Nit: `CHECK` (as this will crash if it fails)

Reply:
Yeah, that makes more sense. I should think more about what should be considered as fatal error.